--- linux-2.6.28-gentoo/fs/Kconfig.orig	2009-01-26 14:03:01.000000000 +0800
+++ linux-2.6.28-gentoo/fs/Kconfig	2009-01-26 18:00:38.000000000 +0800
@@ -726,8 +726,7 @@
 
 config HUGETLBFS
 	bool "HugeTLB file system support"
-	depends on X86 || IA64 || PPC64 || SPARC64 || (SUPERH && MMU) || \
-		   (S390 && 64BIT) || BROKEN
+	depends on X86 || IA64 || PPC64 || SPARC64 || (SUPERH && MMU) || (S390 && 64BIT) || BROKEN
 	help
 	  hugetlbfs is a filesystem backing for HugeTLB pages, based on
 	  ramfs. For architectures that support it, say Y here and read
--- linux-2.6.28-gentoo/lib/Kconfig.debug.orig	2009-01-26 15:15:27.000000000 +0800
+++ linux-2.6.28-gentoo/lib/Kconfig.debug	2009-01-26 18:02:01.000000000 +0800
@@ -465,8 +465,7 @@
 config DEBUG_BUGVERBOSE
 	bool "Verbose BUG() reporting (adds 70K)" if DEBUG_KERNEL && EMBEDDED
 	depends on BUG
-	depends on ARM || AVR32 || M32R || M68K || SPARC32 || SPARC64 || \
-		   FRV || SUPERH || GENERIC_BUG || BLACKFIN || MN10300
+	depends on ARM || AVR32 || M32R || M68K || SPARC32 || SPARC64 || FRV || SUPERH || GENERIC_BUG || BLACKFIN || MN10300
 	default !EMBEDDED
 	help
 	  Say Y here to make BUG() panics output the file name and line number
@@ -547,9 +546,7 @@
 
 config FRAME_POINTER
 	bool "Compile the kernel with frame pointers"
-	depends on DEBUG_KERNEL && \
-		(X86 || CRIS || M68K || M68KNOMMU || FRV || UML || S390 || \
-		 AVR32 || SUPERH || BLACKFIN || MN10300)
+	depends on DEBUG_KERNEL && (X86 || CRIS || M68K || M68KNOMMU || FRV || UML || S390 || AVR32 || SUPERH || BLACKFIN || MN10300)
 	default y if DEBUG_INFO && UML
 	help
 	  If you say Y here the resulting kernel image will be slightly larger
--- linux-2.6.28-gentoo/arch/x86/Kconfig.orig	2009-01-26 18:26:12.000000000 +0800
+++ linux-2.6.28-gentoo/arch/x86/Kconfig	2009-01-26 18:26:35.000000000 +0800
@@ -290,9 +290,9 @@
 	  say N here, otherwise the kernel you build will not be bootable.
 
 config X86_GENERICARCH
-       bool "Generic architecture"
+	bool "Generic architecture"
 	depends on X86_32
-       help
+	help
           This option compiles in the NUMAQ, Summit, bigsmp, ES7000, default
 	  subarchitectures.  It is intended for a generic binary kernel.
 	  if you select them all, kernel will probe it one by one. and will
--- linux-2.6.28-gentoo/drivers/gpu/drm/Kconfig.orig	2009-01-26 13:52:15.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/gpu/drm/Kconfig	2009-01-26 13:53:09.000000000 +0800
@@ -52,7 +52,6 @@
 choice
 	prompt "Intel 830M, 845G, 852GM, 855GM, 865G"
 	depends on DRM && AGP && AGP_INTEL
-	optional
 
 config DRM_I830
 	tristate "i830 driver"
--- linux-2.6.28-gentoo/drivers/ide/Kconfig.orig	2009-01-26 13:27:39.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/ide/Kconfig	2009-01-26 18:09:03.000000000 +0800
@@ -904,7 +904,6 @@
 endif
 
 config BLK_DEV_IDEDMA
-	def_bool BLK_DEV_IDEDMA_SFF || \
-		 BLK_DEV_IDEDMA_ICS || BLK_DEV_IDE_AU1XXX_MDMA2_DBDMA
+	def_bool BLK_DEV_IDEDMA_SFF || BLK_DEV_IDEDMA_ICS || BLK_DEV_IDE_AU1XXX_MDMA2_DBDMA
 
 endif # IDE
--- linux-2.6.28-gentoo/drivers/mfd/Kconfig.orig	2009-01-26 13:50:24.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/mfd/Kconfig	2009-01-26 13:50:42.000000000 +0800
@@ -11,7 +11,7 @@
 
 config MFD_SM501
 	tristate "Support for Silicon Motion SM501"
-	 ---help---
+	---help---
 	  This is the core driver for the Silicon Motion SM501 multimedia
 	  companion chip. This device is a multifunction device which may
 	  provide numerous interfaces including USB host controller, USB gadget,
@@ -22,7 +22,7 @@
 config MFD_SM501_GPIO
 	bool "Export GPIO via GPIO layer"
 	depends on MFD_SM501 && GPIOLIB
-	 ---help---
+	---help---
 	 This option uses the gpio library layer to export the 64 GPIO
 	 lines on the SM501. The platform data is used to supply the
 	 base number for the first GPIO line to register.
@@ -30,7 +30,7 @@
 config MFD_ASIC3
 	bool "Support for Compaq ASIC3"
 	depends on GENERIC_HARDIRQS && GPIOLIB && ARM
-	 ---help---
+	---help---
 	  This driver supports the ASIC3 multifunction chip found on many
 	  PDAs (mainly iPAQ and HTC based ones)
 
--- linux-2.6.28-gentoo/drivers/net/wireless/Kconfig.orig	2009-01-26 13:36:39.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/net/wireless/Kconfig	2009-01-26 15:19:51.000000000 +0800
@@ -443,7 +443,7 @@
       select WIRELESS_EXT
       select FW_LOADER
       select CRC32
-       ---help---
+      ---help---
         A driver 802.11b wireless cards based on the Atmel fast-vnet
         chips. This driver supports standard Linux wireless extensions.
 
@@ -457,7 +457,7 @@
 config PCI_ATMEL
       tristate "Atmel at76c506 PCI cards"
       depends on ATMEL && PCI
-       ---help---
+      ---help---
         Enable support for PCI and mini-PCI cards containing the
         Atmel at76c506 chip.
 
@@ -495,7 +495,7 @@
       tristate "Planet WL3501 PCMCIA cards"
       depends on EXPERIMENTAL && PCMCIA && WLAN_80211
       select WIRELESS_EXT
-       ---help---
+      ---help---
          A driver for WL3501 PCMCIA 802.11 wireless cards made by Planet.
 	 It has basic support for Linux wireless extensions and initial
 	 micro support for ethtool.
--- linux-2.6.28-gentoo/drivers/net/Kconfig.orig	2009-01-26 13:35:29.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/net/Kconfig	2009-01-26 18:27:54.000000000 +0800
@@ -515,9 +515,7 @@
 
 config SH_ETH
 	tristate "Renesas SuperH Ethernet support"
-	depends on SUPERH && \
-		(CPU_SUBTYPE_SH7710 || CPU_SUBTYPE_SH7712 || CPU_SUBTYPE_SH7763 || \
-		 CPU_SUBTYPE_SH7619)
+	depends on SUPERH && (CPU_SUBTYPE_SH7710 || CPU_SUBTYPE_SH7712 || CPU_SUBTYPE_SH7763 || CPU_SUBTYPE_SH7619)
 	select CRC32
 	select MII
 	select MDIO_BITBANG
@@ -893,8 +891,7 @@
 	tristate "SMC 91C9x/91C1xxx support"
 	select CRC32
 	select MII
-	depends on ARM || REDWOOD_5 || REDWOOD_6 || M32R || SUPERH || \
-		MIPS || BLACKFIN || MN10300
+	depends on ARM || REDWOOD_5 || REDWOOD_6 || M32R || SUPERH || MIPS || BLACKFIN || MN10300
 	help
 	  This is a driver for SMC's 91x series of Ethernet chipsets,
 	  including the SMC91C94 and the SMC91C111. Say Y if you want it
@@ -1392,8 +1389,7 @@
 
 config CS89x0
 	tristate "CS89x0 support"
-	depends on NET_ETHERNET && (ISA || EISA || MACH_IXDP2351 \
-		|| ARCH_IXDP2X01 || ARCH_PNX010X || MACH_MX31ADS)
+	depends on NET_ETHERNET && (ISA || EISA || MACH_IXDP2351 || ARCH_IXDP2X01 || ARCH_PNX010X || MACH_MX31ADS)
 	---help---
 	  Support for CS89x0 chipset based Ethernet cards. If you have a
 	  network (Ethernet) card of this type, say Y and read the
--- linux-2.6.28-gentoo/drivers/mtd/maps/Kconfig.orig	2009-01-26 13:22:37.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/mtd/maps/Kconfig	2009-01-26 13:23:00.000000000 +0800
@@ -96,9 +96,7 @@
 
 config MTD_PMC_MSP_RAMROOT
 	tristate "Embedded RAM block device for root on PMC-Sierra MSP"
-	depends on PMC_MSP_EMBEDDED_ROOTFS && \
-			(MTD_BLOCK || MTD_BLOCK_RO) && \
-			MTD_RAM
+	depends on PMC_MSP_EMBEDDED_ROOTFS && (MTD_BLOCK || MTD_BLOCK_RO) && MTD_RAM
 	help
 	  This provides support for the embedded root file system
           on PMC MSP devices.  This memory is mapped as a MTD block device.
--- linux-2.6.28-gentoo/drivers/mtd/nand/Kconfig.orig	2009-01-26 13:23:16.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/mtd/nand/Kconfig	2009-01-26 18:11:34.000000000 +0800
@@ -256,12 +256,12 @@
           will be named "excite_nandflash.ko".
 
 config MTD_NAND_CAFE
-	tristate "NAND support for OLPC CAFÉ chip"
+	tristate "NAND support for OLPC CAFE chip"
 	depends on PCI
 	select REED_SOLOMON
 	select REED_SOLOMON_DEC16
 	help
-	  Use NAND flash attached to the CAFÉ chip designed for the OLPC
+	  Use NAND flash attached to the CAFE chip designed for the OLPC
 	  laptop.
 
 config MTD_NAND_CS553X
--- linux-2.6.28-gentoo/drivers/acpi/Kconfig.orig	2009-01-26 10:09:04.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/acpi/Kconfig	2009-01-26 10:09:46.000000000 +0800
@@ -364,7 +364,7 @@
 	tristate "ACPI0004,PNP0A05 and PNP0A06 Container Driver (EXPERIMENTAL)"
 	depends on EXPERIMENTAL
 	default (ACPI_HOTPLUG_MEMORY || ACPI_HOTPLUG_CPU || ACPI_HOTPLUG_IO)
-	 ---help---
+	---help---
 	  This allows _physical_ insertion and removal of CPUs and memory.
 	  This can be useful, for example, on NUMA machines that support
 	  ACPI based physical hotplug of nodes, or non-NUMA machines that
--- linux-2.6.28-gentoo/drivers/char/Kconfig.orig	2009-01-26 13:44:44.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/char/Kconfig	2009-01-26 13:45:00.000000000 +0800
@@ -765,8 +765,7 @@
 
 config RTC
 	tristate "Enhanced Real Time Clock Support (legacy PC RTC driver)"
-	depends on !PPC && !PARISC && !IA64 && !M68K && !SPARC && !FRV \
-			&& !ARM && !SUPERH && !S390 && !AVR32
+	depends on !PPC && !PARISC && !IA64 && !M68K && !SPARC && !FRV && !ARM && !SUPERH && !S390 && !AVR32
 	---help---
 	  If you say Y here and create a character special file /dev/rtc with
 	  major number 10 and minor number 135 using mknod ("man mknod"), you
--- linux-2.6.28-gentoo/drivers/idle/Kconfig.orig	2009-01-26 10:09:55.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/idle/Kconfig	2009-01-26 10:10:10.000000000 +0800
@@ -1,6 +1,6 @@
 
 menu "Memory power savings"
-depends on X86_64
+	depends on X86_64
 
 config I7300_IDLE_IOAT_CHANNEL
 	bool
--- linux-2.6.28-gentoo/drivers/misc/Kconfig.orig	2009-01-26 13:26:45.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/misc/Kconfig	2009-01-26 18:29:12.000000000 +0800
@@ -289,7 +289,7 @@
 	depends on X86 && ACPI
 	select BACKLIGHT_CLASS_DEVICE
 	depends on INPUT
-	  ---help---
+	---help---
 	  This mini-driver drives the SNC and SPIC devices present in the ACPI
 	  BIOS of the Sony Vaio laptops.
 
@@ -302,7 +302,7 @@
 config SONYPI_COMPAT
 	bool "Sonypi compatibility"
 	depends on SONY_LAPTOP
-	  ---help---
+	---help---
 	  Build the sonypi driver compatibility code into the sony-laptop driver.
 
 config THINKPAD_ACPI
--- linux-2.6.28-gentoo/drivers/hwmon/Kconfig.orig	2009-01-26 13:49:30.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/hwmon/Kconfig	2009-01-26 13:49:44.000000000 +0800
@@ -294,7 +294,7 @@
 	  will be called f71882fg.
 
 config SENSORS_F75375S
-	tristate "Fintek F75375S/SP and F75373";
+	tristate "Fintek F75375S/SP and F75373"
 	depends on I2C && EXPERIMENTAL
 	help
 	  If you say yes here you get support for hardware monitoring
--- linux-2.6.28-gentoo/drivers/media/dvb/frontends/Kconfig.orig	2009-01-26 13:51:47.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/media/dvb/frontends/Kconfig	2009-01-26 13:52:01.000000000 +0800
@@ -54,7 +54,7 @@
 	tristate "ST STB6000 silicon tuner"
 	depends on DVB_CORE && I2C
 	default m if DVB_FE_CUSTOMISE
-	  help
+	help
 	  A DVB-S silicon tuner module. Say Y when you want to support this tuner.
 
 config DVB_STV0299
--- linux-2.6.28-gentoo/drivers/video/Kconfig.orig	2009-01-26 13:54:02.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/video/Kconfig	2009-01-26 18:18:31.000000000 +0800
@@ -298,7 +298,6 @@
 	  will be called amba-clcd.
 
 choice
-
 	depends on FB_ARMCLCD && (ARCH_LH7A40X || ARCH_LH7952X)
 	prompt "LCD Panel"
 	default FB_ARMCLCD_SHARP_LQ035Q7DB02
@@ -1099,7 +1098,6 @@
 	bool "Enable DDC Support"
 	depends on FB_I810 && FB_I810_GTF
 	select FB_DDC
-	help
 
 config FB_LE80578
 	tristate "Intel LE80578 (Vermilion) support"
@@ -2120,11 +2118,11 @@
 source "drivers/video/display/Kconfig"
 
 if VT
-	source "drivers/video/console/Kconfig"
+source "drivers/video/console/Kconfig"
 endif
 
 if FB || SGI_NEWPORT_CONSOLE
-	source "drivers/video/logo/Kconfig"
+source "drivers/video/logo/Kconfig"
 endif
 
 endmenu
--- linux-2.6.28-gentoo/drivers/auxdisplay/Kconfig.orig	2009-01-26 14:01:40.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/auxdisplay/Kconfig	2009-01-26 14:02:03.000000000 +0800
@@ -100,7 +100,7 @@
 	  If unsure, say N.
 
 config CFAG12864B_RATE
-       int "Refresh rate (hertz)"
+	int "Refresh rate (hertz)"
 	depends on CFAG12864B
 	default "20"
 	---help---
--- linux-2.6.28-gentoo/drivers/firmware/Kconfig.orig	2009-01-26 14:02:15.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/firmware/Kconfig	2009-01-26 14:02:42.000000000 +0800
@@ -104,9 +104,9 @@
 	  management software such as Dell OpenManage.
 
 config DMIID
-    bool "Export DMI identification via sysfs to userspace"
-    depends on DMI
-    default y
+	bool "Export DMI identification via sysfs to userspace"
+	depends on DMI
+	default y
 	help
 	  Say Y here if you want to query SMBIOS/DMI system identification
 	  information from userspace through /sys/class/dmi/id/ or if you want
--- linux-2.6.28-gentoo/drivers/pcmcia/Kconfig.orig	2009-01-26 10:10:33.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/pcmcia/Kconfig	2009-01-26 10:11:10.000000000 +0800
@@ -215,9 +215,7 @@
 config PCMCIA_PXA2XX
 	tristate "PXA2xx support"
 	depends on ARM && ARCH_PXA && PCMCIA
-	depends on (ARCH_LUBBOCK || MACH_MAINSTONE || PXA_SHARPSL \
-		    || MACH_ARMCORE || ARCH_PXA_PALM || TRIZEPS_PCMCIA \
-		    || ARCH_VIPER)
+	depends on (ARCH_LUBBOCK || MACH_MAINSTONE || PXA_SHARPSL || MACH_ARMCORE || ARCH_PXA_PALM || TRIZEPS_PCMCIA || ARCH_VIPER)
 	help
 	  Say Y here to include support for the PXA2xx PCMCIA controller
 
--- linux-2.6.28-gentoo/drivers/parport/Kconfig.orig	2009-01-26 13:25:46.000000000 +0800
+++ linux-2.6.28-gentoo/drivers/parport/Kconfig	2009-01-26 13:26:11.000000000 +0800
@@ -35,8 +35,7 @@
 
 config PARPORT_PC
 	tristate "PC-style hardware"
-	depends on (!SPARC64 || PCI) && !SPARC32 && !M32R && !FRV && \
-		(!M68K || ISA) && !MN10300 && !AVR32 && !BLACKFIN
+	depends on (!SPARC64 || PCI) && !SPARC32 && !M32R && !FRV && (!M68K || ISA) && !MN10300 && !AVR32 && !BLACKFIN
 	---help---
 	  You should say Y here if you have a PC-style parallel port. All
 	  IBM PC compatible computers and some Alphas have PC-style
--- linux-2.6.28-gentoo/kernel/power/Kconfig.orig	2009-01-26 18:21:29.000000000 +0800
+++ linux-2.6.28-gentoo/kernel/power/Kconfig	2009-01-26 18:22:18.000000000 +0800
@@ -106,8 +106,7 @@
 	linked, ensuring that it's available when this test runs.
 
 config SUSPEND_FREEZER
-	bool "Enable freezer for suspend to RAM/standby" \
-		if ARCH_WANTS_FREEZER_CONTROL || BROKEN
+	bool "Enable freezer for suspend to RAM/standby" if ARCH_WANTS_FREEZER_CONTROL || BROKEN
 	depends on SUSPEND
 	default y
 	help
--- linux-2.6.28-gentoo/kernel/Kconfig.hz.orig	2009-01-26 10:05:53.000000000 +0800
+++ linux-2.6.28-gentoo/kernel/Kconfig.hz	2009-01-26 10:06:44.000000000 +0800
@@ -18,32 +18,32 @@
 
 	config HZ_100
 		bool "100 HZ"
-	help
-	  100 Hz is a typical choice for servers, SMP and NUMA systems
-	  with lots of processors that may show reduced performance if
-	  too many timer interrupts are occurring.
+		help
+		  100 Hz is a typical choice for servers, SMP and NUMA systems
+		  with lots of processors that may show reduced performance if
+		  too many timer interrupts are occurring.
 
 	config HZ_250
 		bool "250 HZ"
-	help
-	 250 Hz is a good compromise choice allowing server performance
-	 while also showing good interactive responsiveness even
-	 on SMP and NUMA systems. If you are going to be using NTSC video
-	 or multimedia, selected 300Hz instead.
+		help
+		 250 Hz is a good compromise choice allowing server performance
+		 while also showing good interactive responsiveness even
+		 on SMP and NUMA systems. If you are going to be using NTSC video
+		 or multimedia, selected 300Hz instead.
 
 	config HZ_300
 		bool "300 HZ"
-	help
-	 300 Hz is a good compromise choice allowing server performance
-	 while also showing good interactive responsiveness even
-	 on SMP and NUMA systems and exactly dividing by both PAL and
-	 NTSC frame rates for video and multimedia work.
+		help
+		 300 Hz is a good compromise choice allowing server performance
+		 while also showing good interactive responsiveness even
+		 on SMP and NUMA systems and exactly dividing by both PAL and
+		 NTSC frame rates for video and multimedia work.
 
 	config HZ_1000
 		bool "1000 HZ"
-	help
-	 1000 Hz is the preferred choice for desktop systems and other
-	 systems requiring fast interactive responses to events.
+		help
+		 1000 Hz is the preferred choice for desktop systems and other
+		 systems requiring fast interactive responses to events.
 
 endchoice
 
